about.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="./about.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@700;900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Ili Artes - about</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img id="menu-btn" src="./assets/icons/menu-btn-purple.svg" alt="menu button" />
+      </nav>
+      <ul id="menu">
+        <li class="menu-item"><a href="./about.html">About</a></li>
+        <li class="menu-item"><a href="./index.html">Home</a></li>
+      </ul>
+      <img id="close-btn" src="./assets/icons/menu-close.svg" alt="">
+      <div id="logo-container">
+        <a id="logo" href="./index.html"
+          ><img src="./assets/Mobile/about/Logo.svg" alt=""
+        /></a>
+      </div>
+    </header>
+    <main>
+      <section id="about-section">
+        <article id="about-wrapper">
+          <div id="about-header">
+            <h1 class="title">
+              Ili Artes<br /><span>Arte en tus manos!</span>
+            </h1>
+          </div>
+          <hr />
+          <p>
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            individuals who want to work with us on shared projects that advance
+            the cause of the Commons, free culture and open knowledge. So if
+            you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, wikipedians, Open knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together
+          </p>
```

Comment:
- [ ] As mentioned previously pelase make sure to stick to the given color swatches as I noticed the paragraph in the "About" page has a different color from the one shown in the Behance design, I'd also suggest you adjust the font-weight and letter spacing 
- Behnace design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/4e916c4d-87c6-43bf-89e1-0e95eedc5f2a)
- Your design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/c5d972b0-e86e-475f-86ee-f0e33316246f)